src/plugins/vis_type_timeseries/public/application/visualizations/views/timeseries/index.js

```diff
@@ -73,16 +73,20 @@ export const TimeSeries = ({
   const chartRef = useRef();
 
   useEffect(() => {
-    const updateCursor = (_, cursor) => {
-      if (chartRef.current) {
+    const updateCursor = ({ cursor }) => {
+      if (
+        chartRef.current &&
+        cursor &&
+        chartRef.current.chartStore.getState().chartId !== cursor.chartId
```

Comment:
can you please explain why this was changed?

Reply:
I wanted to avoid set cursor for visualization which triggers this event but I checked it, unfortunately it's private property. I'll remove it.